db/schema.rb

```diff
@@ -59,6 +59,7 @@
     t.string   "access_token"
     t.string   "scope"
     t.string   "code_challenge"
+    t.string   "rails_session_id"
```

Comment:
my refactor was a little too aggressive, we're still looking identities up by the `session_uuid`, not by this new `rails_session_id`